go/vt/schemamanager/tablet_executor.go

```diff
@@ -267,12 +263,11 @@ func (exec *TabletExecutor) executeSQL(ctx context.Context, sql string, execResu
 			execResult.ExecutorErr = err.Error()
 			return err
 		}
-		if exec.ddlStrategySetting.IsSkipTopo() {
-			exec.executeOnAllTablets(ctx, execResult, onlineDDL.SQL, true)
-			exec.wr.Logger().Printf("%s\n", onlineDDL.UUID)
-		} else {
-			exec.executeOnlineDDL(ctx, execResult, onlineDDL)
-		}
+		exec.executeOnAllTablets(ctx, execResult, onlineDDL.SQL, true)
+		exec.wr.Logger().Printf("%s\n", onlineDDL.UUID)
```

Comment:
The change here is to remove `if exec.ddlStrategySetting.IsSkipTopo() {` -- we've already hard coded that as `true` is previous releases.

And of course the `else` code cannot ever happen, so that's also removed.